pages/api/admin/update-criminal-check-status.ts

```diff
@@ -0,0 +1,48 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { ResponseUtil } from "@utils/responseUtil";
+import { updateVolunteerCriminalCheckApproval } from "@database/user";
+import { getSession } from "next-auth/client";
+import { roles } from ".prisma/client";
+
+/**
+ * handle controls the request made to the api/admin/criminal-check-update endpoint
+ * @param req API request object
+ * @param res API response object
+ */
+export default async function handle(
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> {
+    const session = await getSession({ req });
+
+    // If there is no session or the user is not a parent, not authorized
+    if (
+        !session ||
+        ![roles.PROGRAM_ADMIN, roles.TEACHER].includes((session as any).role)
+    ) {
```

Comment:
Let's simplify this:
```suggestion
    if (!session || !isAdmin(session)) {
```